src/Styleguide/Pages/Artwork/Sidebar.tsx

```diff
@@ -71,10 +75,29 @@ export class Sidebar extends Component<ArtworkSidebarProps> {
     return (
       <SidebarContainer>
         <Artists artists={artwork.artists} />
+
+        {artwork.is_biddable &&
+          artwork.sale_artwork &&
+          artwork.sale_artwork.lot_label && (
+            <Serif size="2" weight="semibold" color="black100">
+              Lot {artwork.sale_artwork.lot_label}
+            </Serif>
+          )}
         <ArtworkMetadata artwork={artwork} />
-        <Separator />
-        <Commercial artwork={artwork} />
-        <PartnerInfo artwork={artwork} />
+
+        {!artwork.is_in_auction && (
```

Comment:
Instead of the Commertial info artwork from the Auctions should include Bidding Activity component (or whatever we decide to name it). The tricky thing is that in case of regular artwork this commercial info comes before partner info and in case of auction artwork Bidding activity info comes after Auction Partner info.

So doing this check separately here